packages/time/src/semester.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable @typescript-eslint/member-ordering */
+
+export const semesterSortOrder = {
+	Spring: 1,
+	Summer: 2,
+	Fall: 3,
+	Winter: 4,
+};
+
+export type SemesterSeason = keyof typeof semesterSortOrder;
+
+export interface ParsedSemester {
+	year: number;
+	season: SemesterSeason;
+	emoji: string;
+}
+
+export default class Semester {
```

Comment:
Since we're trying to get avoid default exports, let's do this

```ts
export class Semester ...

/** @deprecated */
export default semester;

```